docs/NETWORKING.md

```diff
@@ -118,9 +118,9 @@ distributions, and could be confusing even for experienced Linux users, for exam
 - Network adapter for ethernet port is implemented as a bridge which enslaves the port and takes
   its interface name and MAC address - this way we can then put application interfaces under this
   bridge to implement switch network instance
-- ACLs are implemented by iptables `mangle` table for the most part (not `filter`). Traffic flow
-  that should not be allowed is marked with a "DROP mark" and routed into a dummy `blackhole`
-  interface
+- If flow logging is enabled, ACLs are implemented by iptables `mangle` table for the most
+  part (not `filter`). Traffic flow that should not be allowed is marked with a "DROP mark"
+  and routed into a dummy `blackhole` interface
```

Comment:
Do we document anywhere the motivation for this? (that if IP filter drops the packet they will not be part of the conntrac hence we can not get flow logs for the rejected/dropped flows)?

Might make sense to add that somewhere for completeness (but not introduced by this PR.)

Reply:
I though that I had it mention somewhere but cannot find it right now.
Added short explanation next to this.